.storybook/preview.js

```diff
@@ -1,3 +1,6 @@
+import 'normalize.css';
+import '../src/styles/index.css';
```

Comment:
Some stories were broken because of missing variables. We also had rendering inconsistencies with h5web because `box-sizing: border-box` was missing. This fixes it all.